env/production/config.json

```diff
@@ -110,5 +110,5 @@
   "OIDC_GROUPS_CLAIM": "cognito:groups",
   "SESSION_COOKIE_DOMAIN": "nextstrain.org",
   "GROUPS_DATA_FILE": "groups.json",
-  "RESOURCE_INDEX": "s3://nextstrain-inventories/resources/v4.json.gz"
+  "RESOURCE_INDEX": "s3://nextstrain-inventories/resources/v5.json.gz"
```

Comment:
> I presume [/pathogens](https://nextstrain-s-add-hmpv-vt7cndy3.herokuapp.com/pathogens) isn't working because the resource index got bumped to v5 which hasn't yet been built? I don't understand the versioning there so will leave to @joverlee521 to review that.

Yup, this is not working in the test app because v5.json.gz is not generated yet. Technically this bump is not necessary because it's adding a new pathogen and doesn't change any of the existing data structure.

Reply:
> Technically this bump is not necessary because it's adding a new pathogen and doesn't change any of the existing data structure.

Right - let's drop this from this PR then. I just generated a new (core) index from `master` and can confirm the hmpv datasets are there.

Reply:
> Right - let's drop this from this PR then.

Will do that as part of rebasing on top of the yellow fever changes.

Reply:
okay, rebased on top of master with the yellow fever changes, dropped the manifest version bump commit, force-pushed, ready to rock.

Reply:
> Ah, so -- just for my edification -- why was it needed when I added yellow fever? That was also new and didn't change anything existing.

Ah sorry, I must have been confused for yellow-fever 🤦‍♀️ . Since the old dataset existed, I thought it was updating the existing default.